feat: add questions 027 to 032 for GitHub Copilot practice test (#362)
FidelusAleksander/ghcertified: content/en/questions/copilot/question-029.md

---
title: "Question 029"

> question: "Which of the following can be added as context in Chat to add context to your prompt? (Select three.)"

Reviewer: ```suggestion
question: "Which of the following can be added as context for your prompt in GitHub Copilot Chat? (Select three.)"
```
